### DISCUSSION TITLE
Is there a way to enable renovate "always on" and "open PRs right away"

### DISCUSSION THREAD

**Discussion by m1n9o** [renovatebot/renovate#13384] [Request Help]
Hi all,
As titled, I hope renovate can create PRs right away without "awaiting schedule". How to do this? I didn't figure out a proper config field to do it actually. 
https://docs.renovatebot.com/key-concepts/scheduling/#global-schedule-vs-specific-schedule

Any suggestions?

Best regards,
Mingo

**Comment by rarkins** [comment]
There is no schedule by default. Please describe in more detail what you are trying to achieve
